src/libraries/Common/src/Interop/Windows/Advapi32/Interop.RegQueryValueEx.cs

```diff
@@ -14,39 +14,12 @@ internal static partial class Interop
     internal static partial class Advapi32
     {
         [LibraryImport(Libraries.Advapi32, EntryPoint = "RegQueryValueExW", StringMarshalling = StringMarshalling.Utf16)]
-        internal static partial int RegQueryValueEx(
+        internal static unsafe partial int RegQueryValueEx(
             SafeRegistryHandle hKey,
             string? lpValueName,
             int[]? lpReserved,
```

Comment:
```suggestion
            int* lpReserved,
```
This is always null. We can make it pointer to make the marshaling cheaper.